Description: Fix make clean while target doesn't exists
 Fix failure of make clean due non exists target.
 .
 alsa-cdsp (1.0.0-1~moode~bullseye) unstable; urgency=medium
 .
   * Initial release of debian packing of alsa_cdsp
Author: Bitkeeper <bitkeeper@users.noreply.github.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2021-11-23

--- alsa-cdsp-1.0.0.orig/Makefile
+++ alsa-cdsp-1.0.0/Makefile
@@ -19,4 +19,4 @@ uninstall:
 	rm $(LIBDIR)/alsa-lib/$(TARGET).so

 clean:
-	rm $(TARGET).so
+	rm -f $(TARGET).so
